.github/workflows/auto-update-otel-sdk.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       latest-version: ${{ steps.check-versions.outputs.latest-version }}
       already-opened: ${{ steps.check-versions.outputs.already-opened }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

Comment:
Wait what? A hash?

Reply:
it looks a bit weird, but does it matter? dependabot used `@v4` for the same update, we could do this update manually or we could let renovate use the hash. Imo as long as renovate keeps it up to date it isn't really important whether it uses the hash or the tag.

Reply:
I think I'd rather bump it to `v4` manually, without mentioning the hash. These files should be readable/maintainable by humans as well, not just renovate.

Reply:
I used "If you want to rebase/retry this PR, check this box", and it picked up the good tag name now